app/config/Constants.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+object Constants {
+
+  object ModeOfTransport {
+    val Maritime = "1"
+    val Rail     = "2"
+    val Road     = "3"
+    val Air      = "4"
+    val Mail     = "5"
+    val Fixed    = "7"
+    val Unknown  = "9"
```

Comment:
Personally don't mind if we just hard code a "3". In the tests we're only really bothered if it is a 3 or any other string